z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -220,7 +220,7 @@ The ansible playbooks has automated origin complex UPI mannul process, such as c
 
 ### Config HAproxy and DNS on a bastion node (default)
 
-User can choose to use your own bastion node to deploy cluster. If there aren't DNS server and Haproxy on bastion node, highly recommend installing them by ansible playbook [Configure Bastion](#configure-bastion), or you have an external DNS server which is managed in other DNS provider, if so, you need to add openshift recorder in your DNS server manually, please see the following [example](#sample-dns-zone-database) to add it.
+User can choose to use your own bastion node to deploy cluster. If there aren't DNS server and Haproxy on bastion node, highly recommend installing them by ansible playbook [Configure Bastion](#configure-bastion), or you have an external DNS server which is managed in other DNS provider, if so, you need to add openshift recorder in your DNS server and HAProxy manually, please see the following [example](#sample-dns-zone-database) to add it.
```

Comment:
Users' load-balancer may not be HAProxy, such as nginx, F5 and so on. So it's better to `and HAProxy` -> `and Load-balancer`

Reply:
Update it.